build/website_cron_testing/README.md

```diff
@@ -0,0 +1,17 @@
+# Website Cron Testing Docker Image
+
+## Description
+
+This is a Docker image based on python:3.11, but comes with:
+
+- Google Chrome: the browser which is used to run the tests.
+- ChromeDriver: used to send commands to Google Chrome.
+- Script (and files required by the script) to run the website cron tests
+
```

Comment:
If you are able to use gcr.io/datcom-ci/webdriver-chrome:2023-06-17, can update here as well

Reply:
done